src/ophyd_epics_devices/zebra.py

```diff
@@ -0,0 +1,306 @@
+from enum import Enum
+from functools import partialmethod
+from typing import List, Type
+
+import numpy as np
+import numpy.typing as npt
+from ophyd.v2.core import StandardReadable, T
+from ophyd.v2.epics import EpicsSignalR, EpicsSignalRW
+
+
+def zebra_rw(datatype: Type[T], suffix: str) -> EpicsSignalRW[T]:
+    return EpicsSignalRW(datatype, suffix)
+
+
+def zebra_r(datatype: Type[T], suffix: str) -> EpicsSignalR[T]:
+    return EpicsSignalR(datatype, suffix)
```

Comment:
Nit: These seem overkill, is it to do the timeouts?

Reply:
Nope, this is something I copied from the area detector device. They were useful there because the read_pv and write_pv were different for the EpicsSignalRW. But here, that is not the case... 

You are right, They do seem a bit much. I'll get rid of them in the next commit.